packages/@lwc/shared/src/aria.ts

```diff
@@ -56,3 +65,77 @@ export const AriaPropertyNames = [
     'ariaValueText',
     'role',
 ];
+
+// We should eventually use the typescript Pick utility to reuse ambient Aria property types when
```

Comment:
I don't think we should do this here, since `@lwc/shared` is not meant to be compiled with the dom lib. Let's remove this comment.